libraries/helpers.rb

```diff
@@ -7,11 +7,21 @@
 #
 def wait_for_block_dev(path)
   Timeout.timeout(60) do
-    sleep(1) until ::File.blockdev?(path)
+    until ::File.blockdev?(path)
+      sleep(5)
```

Comment:
Bumped up the sleep between retries to minimize the number of times we rescan the bus. Scanning each second seems a bit wasteful.